src/Campaigns/Routes/RegisterCampaignRoutes.php

```diff
@@ -0,0 +1,305 @@
+                        'minimum' => 1,
+                    ],
+                    'per_page' => [
+                        'type' => 'integer',
+                        'default' => 30,
+                        'minimum' => 1,
+                        'maximum' => 100,
+                    ],
+                ],
+            ]
+        );
+    }
+
+    /**
+     * Update Campaign route
+     *
+     * @unreleased
+     */
+    public function registerUpdateCampaign()
+    {
+        register_rest_route(
+            CampaignRoute::NAMESPACE,
+            CampaignRoute::CAMPAIGN,
+            [
+                [
+                    'methods' => WP_REST_Server::EDITABLE,
+                    'callback' => function (WP_REST_Request $request) {
+                        return $this->campaignRequestController->updateCampaign($request);
+                    },
+                    'permission_callback' => function () {
+                        return current_user_can('manage_options');
+                    },
+                ],
+                'args' => [
+                    'id' => [
+                        'type' => 'integer',
+                        'required' => true,
+                    ],
+                    'title' => [
+                        'type' => 'string',
+                        'sanitize_callback' => 'rest_sanitize_request_arg',
+                        'validate_callback' => 'rest_validate_request_arg'
+                    ],
+                ],
+                'schema' => [$this, 'getSchema'],
+            ]
+        );
+    }
+
+
+    /**
+     * Create Campaign route
+     *
+     * @unreleased
+     */
+    public function registerCreateCampaign()
+    {
+        register_rest_route(
+            CampaignRoute::NAMESPACE,
+            CampaignRoute::CAMPAIGNS,
+            [
+                [
+                    'methods' => WP_REST_Server::CREATABLE,
+                    'callback' => function (WP_REST_Request $request) {
+                        return $this->campaignRequestController->createCampaign($request);
+                    },
+                    'permission_callback' => function () {
+                        return current_user_can('manage_options');
+                    },
+                ],
+                'args' => [
+                    'title' => [
+                        'type' => 'string',
+                        'required' => true,
+                        'sanitize_callback' => 'sanitize_text_field',
+                    ],
+                    'description' => [
+                        'type' => 'string',
+                        'required' => false,
+                        'sanitize_callback' => 'sanitize_text_field',
+                    ],
+                    'startDateTime' => [
+                        'type' => 'string',
+                        'format' => 'date-time', // @link https://datatracker.ietf.org/doc/html/rfc3339#section-5.8
+                        'required' => true,
```

Comment:
As this is not required on the UI, let's replace this line with this one: `'required' => false,`